oarepo_ui/templates/components/SidebarApiLinks.jinja

```diff
@@ -1,7 +1,8 @@
 {# def links #}
 <div class="ui link list">
     {% for l in links.items() %}
-        {% if "/api" in l.1 %}
+        {# to not show draft link in published record detail page#}
```

Comment:
Does this show draft links on draft record preview page however?

Reply:
@mirekys We don't have a preview page where you can see the draft version of the record at this point. Though we will definitely have it in the near future. This is more a bit of a patch, in order to not show a broken link in the detail page right now, because I accidentally saw it is not working. I think that, if possible, this should be solved in the BE code that generates the links, i.e. that different sets of links would be generated for a published record and a draft and in which case such filtering would not be needed on the FE. Maybe worth discussing this with BE guys?

Reply:
This should probably be tracked in: #EP-51